ui/src/index.css

```
@@ -2982,36 +2982,324 @@ table.auto-table {
+}
+
+/* 影院/全屏看盘 — 覆盖全屏 Overlay (白色极简全屏 Mode) */
+.game-fs-overlay {
   position: fixed;
   inset: 0;
-  z-index: 2000;
-  background: rgb(0 0 0 / 60%);
+  width: 100vw;
+  height: 100vh;
+  z-index: 99999;
+  background: #ffffff;
```

Stylelint (17.14.0)

[error] Expected "`#ffffff`" to be "`#fff`" (color-hex-length)

(color-hex-length)